src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/InnvilgeBarnetilsyn.tsx

```diff
@@ -92,6 +103,20 @@ export const InnvilgeBarnetilsyn: React.FC<Props> = ({ lagretVedtak, barnIBehand
         return form;
     };
 
+    const beregnBarnetilsyn = () => {
+        if (formState.customValidate(validerPerioder)) {
```

Comment:
Kan være greit å legge til håndtering av dobbelklikk, selvom det ikke er så viktigere for lagring

Reply:
Endret: Håndterer at man er i status laster både ved beregning og lagring, som hindrer å kunne lagre (dobbeltklikk). Man kan dobbelttrykke på beregning uten problemer